impl/src/main/resources/org/jboss/cdi/tck/tests/context/passivating/enterprise/valid/ejb-jar.xml

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<ejb-jar
```

Comment:
`org.jboss.cdi.tck.tests.context.passivating.enterprise.valid.StatefulSessionBeanXmlDescriptorTest` seems to be using this, see https://github.com/eclipse-ee4j/cdi-tck/blob/master/web/src/main/java/org/jboss/cdi/tck/tests/context/passivating/enterprise/valid/StatefulSessionBeanXmlDescriptorTest.java#L47
So we will still need this, only under new package/path.

Reply:
Note `impl` vs. `web`. These files are only needed in `web`, and this PR tries to remove them from `impl`, which is OK if you ask me.

Reply:
Ah my bad, didn't notice that there is an equivalent file in `web`.